extension/src/background/messageListener/popupMessageListener.ts

```diff
@@ -1493,12 +1494,7 @@ export const popupMessageListener = (request: Request, sessionStore: Store) => {
         await server.submitTransaction(builtTransaction);
       } catch (e) {
         console.error(e);
-        migrationErrors.push({
-          source: publicKey,
-          destination: newKeyPair.publicKey,
-          error: e,
-        });
-        break;
+        migratedAccount.isMigrated = false;
```

Comment:
if any part of the migration fails, we'll tag it as !isMigrated and continue the process